es-ds-docs/components/ds-molecules-list.vue

```diff
@@ -1,59 +1,40 @@
 <template>
```

Comment:
Looks like a lot of changes here, is this from a formatter?

Reply:
Yeah, I meant to add a note about that. I just added a list item for Modal and I haven't changed anything about my VSCode configuration in a while.

Is this list of [recommended extensions](https://github.com/EnergySage/es-ds/blob/main/.vscode/extensions.json) for es-ds still current? I noticed Vetur in there.

Reply:
Ahh yeah, I should create a ticket to update that. Vetur is for vue2, for the esds-3.0 branch we should be using https://marketplace.visualstudio.com/items?itemName=Vue.volar

Reply:
I disabled `esbenp.prettier-vscode` it didn't try to reformat that file. Are we still using that one? It's in the list of extensions, but I thought I saw a comment from Ryan or someone else about it being obsolete.

Reply:
After reviewing all my VSCode extensions and their settings I don't understand why Prettier keeps reformatting that file, unless that's the way it should be. I disabled the extension to add my changes for `Modal` and I won't touch it again.